src/streamlink/plugins/zattoo.py

```diff
@@ -175,21 +184,31 @@ def _hello(self):
         params = {
             'client_app_token': app_token,
             'uuid': __uuid,
-            'lang': 'en',
-            'format': 'json'
         }
-        res = self.session.http.post(hello_url, headers=self.headers, data=params)
+
+        if self.base_url == 'https://www.quantum-tv.com':
+            params['app_version'] = '3.2028.3'
+        else:
+            params['lang'] = 'en'
+            params['format'] = 'json'
+
+        res = self.session.http.post(hello_url, headers=self.headers, data=params, cookies=self.session.http.cookies)
 
     def _login(self, email, password):
         log.debug('_login ... Attempting login as {0}'.format(email))
 
-        login_url = self.API_LOGIN.format(self.base_url)
         params = {
             'login': email,
             'password': password,
             'remember': 'true'
         }
 
+        if self.base_url == 'https://quantum-tv.com':
+            login_url = self.API_LOGIN_V3.format(self.base_url)
+            params['remember'] = 'True'
```

Comment:
it is already there in line 203, is it with a upper case now instead of a lower case?

Reply:
they were in some cases extremely picky about the things they accept, so I used the same values as the webapp
but it works also with a lower case True